### ISSUE TITLE
Horizontal Scrolling

### ISSUE THREAD

**Issue by finnp** [douglasjunior/react-native-pdf-renderer#12]
Hey,

I was wondering if you are open to adding horizontal scrolling to this module or if you want to keep it simple?

I'm asking because with PdfKit these things are quite trivial, but on Android it would require a few more adjustments.

Best,
Finn

**Comment by douglasjunior** [comment]
Yeah, probably this will require a lot of work because of zoom of the RecyclerView.

**Comment by tomelsner** [comment]
+1 for horizontal scrolling. Anything we can help to make this happen?

**Comment by douglasjunior** [comment]
I believe we'll need to do the following:

- Change the layout orientation: https://github.com/douglasjunior/react-native-pdf-renderer/blob/d3ba0cf1b96118f8899b79b4a5cb5ab1976d6e9c/android/src/main/java/com/github/douglasjunior/reactNativePdfRenderer/modules/PdfRendererRecyclerView.java#L79
- Update the page size and margin calculation: https://github.com/douglasjunior/react-native-pdf-renderer/blob/d3ba0cf1b96118f8899b79b4a5cb5ab1976d6e9c/android/src/main/java/com/github/douglasjunior/reactNativePdfRenderer/modules/PdfRendererRecyclerView.java#L427-L433
- Adjust horizontal scrolling to manage the scroll speed when zoomed in: https://github.com/douglasjunior/react-native-pdf-renderer/blob/d3ba0cf1b96118f8899b79b4a5cb5ab1976d6e9c/android/src/main/java/com/github/douglasjunior/reactNativePdfRenderer/modules/PdfRendererRecyclerView.java#L460 
- Perform regression tests to ensure that everything is working properly and nothing is broken.

**Comment by fefeLirr** [comment]
Hi, please can it support text reflow ? If yes how can it be done ?
